src/core/server/rendering/views/fonts.tsx

```diff
@@ -230,11 +365,16 @@ export const Fonts: FunctionComponent<Props> = ({ url }) => {
     ],
   };
 
+  /**
+   * If `themeVersion` is not provided, we want to fallback to the newest font family `Inter`
+   */
+  const interFont = themeVersion === 'v7' ? interUi : inter;
```

Comment:
We can avoid instantiating both fonts to only use one by extracting the `FontFace` generation logic

```ts
const interFont = themeVersion === 'v7' ? getInterUi(url) : getInter(url);
```

Reply:
Done! https://github.com/elastic/kibana/pull/102359/commits/91ea2e1337d919f1c2a89c7a9b1f8011cda41362